### PR TITLE
Handle tgz

### PR DESCRIPTION
**Pull request by hannah98**
This PR addresses Issue #27

Changes in this PR:

1. 2 Tests were failing.  Updated all tests to passing before making changes to main code
2. Added a test to check if a .tgz file is an archive and expect it to be true
3. Changed the `isArchiveFile` function to include `.tgz` as a valid archive extension
4. All tests passing

I don't know anything about Golang so feel free to correct anything I did.

### CODE REVIEW

File: lib/util.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 
 func isArchiveFile(filePath string) bool {
 	fileExtension := filepath.Ext(filePath)
-	if fileExtension == ".br" || fileExtension == ".bz2" || fileExtension == ".zip" || fileExtension == ".gz" || fileExtension == ".lz4" || fileExtension == ".sz" || fileExtension == ".xz" || fileExtension == ".zst" || fileExtension == ".tar" || fileExtension == ".rar" {
+	if fileExtension == ".br" || fileExtension == ".bz2" || fileExtension == ".zip" || fileExtension == ".gz" || fileExtension == ".lz4" || fileExtension == ".sz" || fileExtension == ".xz" || fileExtension == ".zst" || fileExtension == ".tar" || fileExtension == ".rar" || fileExtension == ".tgz" {
 		return true
 	}
 	return false
```
Comment (marwanhawari) [inline lib/util.go:22]: Love the thought that this should be comprehensive! I actually looked again into the archiver (v3) library and found [this method](https://github.com/mholt/archiver/blob/v3.5.1/archiver.go#L441) `ByExtension` that actually does what we are looking for! Might I suggest the following:

```diff
func isArchiveFile(filePath string) bool {
-	fileExtension := filepath.Ext(filePath)
-	if fileExtension == ".br" || fileExtension == ".bz2" || fileExtension == ".zip" || fileExtension == ".gz" || fileExtension == ".lz4" || fileExtension == ".sz" || fileExtension == ".xz" || fileExtension == ".zst" || fileExtension == ".tar" || fileExtension == ".rar" || fileExtension == ".tgz" {
-		return true
-	}
-	return false
}

func isArchiveFile(filePath string) bool {
+	_, err := archiver.ByExtension(filePath)
+	return err == nil
}

**Comment by marwanhawari** [comment]
Thank you for fixing old tests and adding new ones!

**Comment by hannah98** [comment]
I apologize, I was out of town for a week.  The new method you used is great!

**Comment by marwanhawari** [comment]
No need to apologize! Thank you for the contribution!

**Pull request merged**